content-src/asrouter/schemas/panel/cfr-fxa-bookmark.schema.json

```diff
@@ -121,6 +121,33 @@
         }
       },
       "required": ["tooltiptext"]
+    },
+    "close_button": {
+      "type": "object",
+      "description": "The small dissmiss icon displayed in the top right corner of the message. Not configurable, only the tooltip text." ,
+      "properties": {
+        "tooltiptext": {
+          "oneOf": [
+            {
+              "allOf": [
+                {"$ref": "#/definitions/plainText"},
+                {"description": "Message to be shown"}
+              ]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "string_id": {
+                  "type": "string",
+                  "description": "Fluent id of localized string"
+                }
+              },
+              "required": ["string_id"]
+            }
+          ]
+        }
+      },
+      "required": ["tooltiptext"]
```

Comment:
I deviated from the schema in Bug 1543121. I can submit a separate PR for my changes but here's the remote message I was using for testing:

```json
{
  "data": {
    "id": "fxa-bookmark-sync",
    "content": {
      "title": "cfr-doorhanger-bookmark-fxa-header",
      "text": "cfr-doorhanger-bookmark-fxa-body",
      "cta": "cfr-doorhanger-bookmark-fxa-link-text",
      "color": "white",
      "background_color_1": "#7d31ae",
      "background_color_2": "#5033be",
      "info_icon": {
        "tooltiptext": "cfr-bookmark-tooltip-text"
      }
    },
    "trigger": {"id": "bookmark-panel"},
    "targeting": "usesFirefoxSync == false && totalBookmarksCount > 10 && localeLanguageCode == 'en'"
  }
}
```